### PR TITLE
Spark+Infinispan simple demo

### PR DESCRIPTION
**Pull request by gustavonalle**
Works doing ```mvn install``` followed by ```mvn exec:exec``` as long as docker is available. 
Support to run it on windows is coming (some day)

### CODE REVIEW

File: spark/src/main/java/org/infinispan/tutorial/simple/spark/SimpleSparkJob.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+   public static void main(String[] args) throws UnknownHostException {
+      // Obtain the local address
+      String localAddress = InetAddress.getLocalHost().getHostAddress();
+
+      // Adjust log levels
+      Logger.getLogger("org").setLevel(Level.WARN);
+
+      // Create the remote cache manager
+      Configuration build = new ConfigurationBuilder().addServer().host(localAddress).build();
```
Comment (tristantarrant) [inline spark/src/main/java/org/infinispan/tutorial/simple/spark/SimpleSparkJob.java]: I like this local address thing, maybe I should change my remote / scripting examples too

File: spark/pom.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <artifactId>infinispan-simple-tutorials</artifactId>
+    </parent>
+    <name>Infinispan Simple Tutorials: Apache Spark integration</name>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>1.4.0</version>
```
Comment (tristantarrant) [inline spark/pom.xml]: the version is defined in the parent pom

**Comment by gustavonalle** [comment]
Updated

**Comment by tristantarrant** [comment]
Merged, thanks

**Pull request closed without merging**